### ISSUE TITLE
Eager to get the code

### ISSUE THREAD

**Issue by guanjunwu** [kaichen-z/DynPoint#1]
Hi, Thanks for your nice work!
I am eager to study your paper with the colmap, especially for getting refined poses and depth bias referred to in your paper!
btw, I really want to know how can you generate depth by colmap? In my implementation, images in `nsff` dataset will be reshaped to a new shape (960*540->969*540) by `image_undistorter`
Do you generate the depth image by the resolution of 969*540 directly? 
Refining depth and pose at the same time seems time-comsuming. How long do you take?

**Comment by kaichen-z** [comment]
Thank you for your interest. The required code will be made available by mid-January 2024.

**Comment by guanjunwu** [comment]
Hi,
waiting for your code!

**Comment by kaichen-z** [comment]
Thank you for your understanding. 

I am pleased to announce that we have now made the code for the depth estimation and scene flow estimation parts of our paper available. However, we have made modifications to the pose estimation part and have decided not to include it in our final code version. The reason for this decision is that it did not consistently perform well across different datasets. That is also why we have not made any modifications to the final version submitted. 

We appreciate your understanding of this decision.
